src/unstructured_client/_hooks/custom/form_utils.py

```diff
@@ -35,7 +35,7 @@ def get_page_range(form_data: FormData, key: str, max_pages: int) -> tuple[int,
     try:
         _page_range = form_data.get(key)
 
-        if _page_range is not None:
+        if _page_range is not None and isinstance(_page_range, list):
```

Comment:
since `isinstance(None, list) == False` -> this can be shorted to just `isinstance(_page_range, list)`

Reply:
this applies to the logics below as well; `None` is its own type.